MH4U Dex/CoreDataController.m

```diff
@@ -55,20 +55,20 @@ - (void)tryLoadSequence
     if ([[NSUserDefaults standardUserDefaults] boolForKey:NeedsReloadKey]) {
         NSLog(@"Resetting Core Data");
         [self resetCoreData];
-        NSLog(@"Core Data Reset. Loading Data.");
-        [self loadMonsterData];
-        NSLog(@"Monster data loaded.");
-        //TODO: Load Damage Zone Data.
-        [self loadMonsterDropData];
-        NSLog(@"Monster Drop data loaded.");
-        [self loadRegionData];
-        NSLog(@"Region data loaded.");
-        [self loadQuestData];
-        NSLog(@"Quest Data loaded.");
-        [self loadQuestDropData];
-        NSLog(@"Quest Drop Data loaded.");
-        [self saveContext];
-        NSLog(@"Core Data Context Saved.");
+//        NSLog(@"Core Data Reset. Loading Data.");
```

Comment:
Yes, I'd like to keep it around for future model changes.  I'll add a TODO to use again when changing the model